README.md

```diff
@@ -54,22 +48,35 @@ Using this, you get this analysis FOR FREE (and it's fast!)
 
 - Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
 
+## <span id="engineering-hints">Engineering and Process</span>
```

Comment:
nit: I think the H2 should automatically give this section an `id`. Not sure if you need to create a second/nested `id`, unless this is linked to from elsewhere like an external repo.